sdk/ml/azure-ai-ml/tests/dsl/unittests/test_dsl_pipeline.py

```diff
@@ -2581,3 +2581,19 @@ def root_pipeline():
         pipeline_job.settings.default_compute = "cpu-cluster"
         validate_result = pipeline_job._validate()
         assert validate_result.error_messages == {}
+
+    def test_dsl_pipeline_component_early_available_property(self):
+        component_func = load_component("./tests/test_configs/components/streaming_component_basic.yml")
+
+        @dsl.pipeline
+        def pipeline_component_func():
+            streaming_node = component_func()
+            return streaming_node.outputs
+
+        @dsl.pipeline
+        def root_pipeline_func():
```

Comment:
What's the behavior when the pipeline has 1 layer? e.g. directly submit the pipeline_component_func() object.

Reply:
Just verify for this, it remains `is_control` and `early_available`, same as in pipeline component.